matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/auth/registration/RegistrationAvailability.kt

```diff
@@ -0,0 +1,8 @@
+package org.matrix.android.sdk.api.auth.registration
+
+import org.matrix.android.sdk.api.failure.Failure
+
+sealed class RegistrationAvailability {
+    data class Available(val available: Boolean): RegistrationAvailability()
```

Comment:
According to https://matrix.org/docs/spec/client_server/r0.6.1#get-matrix-client-r0-register-available, the boolean is useless, I think you can remove it.

Reply:
Well, I cant leave data class empty, so I would delete the hole `Available ` class. That mean `registerAvailable` endpoint would return `Any`, and Im not sure if its a good practice?

Reply:
Well, I cant leave data class empty